apps/api/src/applicant/ienapplicant.controller.ts

```diff
@@ -59,6 +59,8 @@ export class IENApplicantController {
   @Get('/')
   async getApplicants(@Query() filter: IENApplicantFilterAPIDTO): Promise<[ApplicantRO[], number]> {
     try {
+      this.logger.log(`list applicant called with the below filter parameters`);
```

Comment:
Logging filter might be good for debugging but not provide any useful information for audit. Also sometimes consecutive loggings are apart from each other when they are actually written to a file.

Reply:
agree with your point, What do you suggest. is removing filter param fine?